src/main/java/teammates/ui/webapi/action/UpdateInstructorAction.java

```diff
@@ -69,7 +69,9 @@ public ActionResult execute() {
                                 .withRole(instructorToEdit.role)
                                 .build());
             }
-            return new JsonResult(new InstructorData(updatedInstructor));
+            InstructorData newInstructorData = new InstructorData(updatedInstructor);
+            newInstructorData.setGoogleId(updatedInstructor.getGoogleId());
+            return new JsonResult(newInstructorData);
```

Comment:
This might actually count as Google ID leaking, but I guess someone who has the privilege of updating instructor has no harm of knowing the Google ID.